### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by fghaas**
Couple of updates to the README:

* Remind users to rebuild the `openedx` image when deploying with Tutor (addresses #232).
* Drop references to the old "native" installation.

### CODE REVIEW

**Comment by fghaas** [comment]
@mrtmm and @foadlind, I've squashed this PR into one commit. I think this is good to merge now, please review.

File: Changelog.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Unreleased
   The asgi application now also checks for allowed hosts,
   meaning if you want to allow the LMS to connect to labs via the `hastexo_guacamole_client`, the LMS host has to be listed in
   `ALLOWED_HOSTS` at `hastexo_guacamole_client.settings.py`.
+  [Documentation] Remove obsolete deployment instructions for the
```
Comment (foadlind) [inline Changelog.md:14]:
```suggestion
  * [Documentation] Remove obsolete deployment instructions for the
```

File: README.md [hunk 2]
```diff
@@ -50,238 +50,99 @@ Course authors can fully define and customize the lab environment. It is only
...
-    - "hastexo-xblock>=6"
-  ```
-  For additional information, please refer to the [official documentation](https://docs.tutor.overhang.io/configuration.html?highlight=xblock#installing-extra-xblocks-and-requirements)
+   `OPENEDX_EXTRA_PIP_REQUIREMENTS` list in `config.yml`:
+   ```
+   OPENEDX_EXTRA_PIP_REQUIREMENTS:
+     - "hastexo-xblock>=6"
+   ```
+   For additional information, please refer to the [official
+   documentation](https://docs.tutor.overhang.io/configuration.html?highlight=xblock#installing-extra-xblocks-and-requirements).
```
Comment (foadlind) [inline README.md:65]: Remove the highlighting of the word "xblock" from the docs page:
```suggestion
   documentation](https://docs.tutor.overhang.io/configuration.html#installing-extra-xblocks-and-requirements).
```

File: README.md [hunk 3]
```diff
@@ -50,238 +50,99 @@ Course authors can fully define and customize the lab environment. It is only
...
-
+3. Before deploying services with Tutor, build the Docker image for
+   the `hastexo` service, and also build a custom `openedx` image:
+   ```
+   tutor images build openedx hastexo
+   ```
+   For more details about each setting, please refer to the *XBlock
+   settings* section below.
+   For more information about the plugin configurations, please refer to the
+   plugin README.
```
Comment (foadlind) [inline README.md:140]: This section should probably be at the end of step number 2 above.

**Review by foadlind — COMMENTED** [review]
Looks good. I have some small suggestions:

File: Changelog.md (hunk 1 above)
Comment (fghaas) [reply Changelog.md:14]: Ah, yes of course. :)

File: README.md (hunk 2 above)
Comment (fghaas) [reply README.md:65]: Yeah, fair point.

File: README.md (hunk 3 above)
Comment (fghaas) [reply README.md:140]: The first sentence, yes. The second one, I think, belongs where it is.

**Review by mrtmm — APPROVED** [review]

**Review by foadlind — APPROVED** [review]
Looks good now. Thanks!

**Comment by fghaas** [comment]
Hmmm I just thought of something else: the "Usage in Devstack" section is obsolete too, right? Should we just drop that, too?

**Comment by mrtmm** [comment]
Hmm, I guess so yes, following the tutor-contrib-hastexo docs will allow installing it in a tutor devstack also.

**Comment by fghaas** [comment]
Ack, see #236.

**Pull request merged**